### ISSUE TITLE
Data Link: any following a data link cancels Kiosk mode

### ISSUE THREAD

**Issue by baur** [grafana/grafana#35691]
**What happened**:

Any data link cancels Kiosk mode, 
Works if only add &kiosk to link, in previous version works wihtout &kiosk 

**What you expected to happen**:

is this feature or bug?

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana version: v8.0.2 (5c5d45e19f)
- Data source type & version: MS SQL
- OS Grafana is installed on: Windows 10
- User OS & Browser: Chrome
- Grafana plugins: 
- Others:

**Comment by marefr** [comment]
Confirmed bug locally using the gdev dashboard "Datalinks - variables" and clicking on data link lower right panel. Haven't compared with v7.x version though.

**Comment by jledun** [comment]
I have the same problem when I'm using custom timeRange with I panel I've built.

When I select kiosk mode with the help of the top right screen icon button then app appends "&kiosk" in the query string parameters.

When I update timeRange, the query string parameters become "&kiosk=" without value => the app goes to normal mode, kiosk mode goes off.

If I set kiosk mode by adding by myself "&kiosk=1" to query string parameters then the bug disappears and the app stays in kiosk mode when the timeRange is updated.

I hope it helps...
